distribution/docs/src/main/resources/content/_reference/_tables/TransformerBlacklist.adoc

```diff
@@ -0,0 +1,33 @@
+:title: Transformer Blacklist Configuration
+:id: org.codice.ddf.catalog.ui.transformer.blacklist
+:type: table
+:status: published
+:application: ${ddf-catalog}
+:summary: Transformer blacklist configuration.
+
+.[[_org.codice.ddf.catalog.ui.transformer.blacklist]]Catalog UI Search Transformer Blacklist
+[cols="1,1m,1,3,1m,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Metacard Transformer Blacklist
+|blackListedMetacardTransformerIds
+|String
+|The IDs of all metacard transformers that will not show up as export actions in the UI.
```

Comment:
✏️ Mention what id means for these two lists. Maybe link to the id part in _transformers-query-response.adoc_.

Reply:
I've added what ID means for these two lists. We currently don't have documentation for metacard transformers so I'm going to leave out the link to the id part each transformer type.